rooms-quickstart-java/pom.xml

```diff
@@ -26,6 +26,11 @@
   </properties>
   <dependencyManagement>
     <dependencies>
+      <dependency>
+          <groupId>com.azure</groupId>
+          <artifactId>azure-core</artifactId>
+          <version>1.48.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
```

Comment:
the comment here  `<!-- {x-version-update;com.azure:azure-core;dependency} -->`, i think this is used in the SDK repo for managing dependency versions from a central versioning file. I don't think the central version managing is used in this quickstarts repo. Can we remove the comment? And also the one a couple lines down?